setup.cfg

```diff
@@ -56,6 +56,7 @@ exclude =
 dev =
     flake8==5.0.4
     isort
+    kornia-rs==0.0.5
```

Comment:
Now that I see this, we should have that dependency such as `pip install kornia[io]`

besides, planning to start adopting [pangolin](https://github.com/stevenlovegrove/Pangolin) soon, since we can provide from there a more wide ecosystem tools @stevenlovegrov what would be the minimum interface required to adopt pangolin here ? Ideally via dlpack so that we have zero copy overhead between drivers and tensors or whatever data structure we want to support here in python for images

Reply:
Opening a separated discussion thread